pandas/tests/tools/test_to_datetime.py

```diff
@@ -2530,3 +2530,16 @@ def test_empty_string_datetime_coerce__unit():
     # verify that no exception is raised even when errors='raise' is set
     result = to_datetime([1, ""], unit="s", errors="raise")
     tm.assert_index_equal(expected, result)
+
+
+def test_to_datetime_monotonic_increasing_index():
+    # GH28238
+    # Create date range of 1000 hour periods
```

Comment:
The comments aren't necessary (except the gh reference)

Reply:
Removed